tests/network/hotplug_bridge.go

```diff
@@ -303,6 +303,26 @@ var _ = SIGDescribe("bridge nic-hotunplug", func() {
 				Expect(err).NotTo(HaveOccurred())
 				assertInterfaceUnplugedFromVM(g, vm, linuxBridgeNetworkName2)
 			}, 30*time.Second, 3*time.Second).Should(Succeed())
+
+			By("Unplug the last secondary interface")
+			Expect(removeInterface(vm, linuxBridgeNetworkName1)).To(Succeed())
+
+			if plugMethod == migrationBased {
+				migrate(vmi)
+			}
+
+			By("verify unplugged iface cleared from VM & VMI")
+			Eventually(func(g Gomega) {
```

Comment:
Can you please export the two `Eventually` blocks to a single method?

Reply:
Done